cli/popper/exceptions.py

```diff
@@ -1,6 +1,6 @@
 import click
-from click._compat import get_text_stderr
 from click import secho
+from click._compat import get_text_stderr
```

Comment:
another thing we can use to simplify the code is to remove this file (`exceptions.py`) as only `cli.py` uses it to throw a `UsageError` exception, which can be changed for a `ClickException`.

Reply:
@ivotron Sure ! Also this PR is still not ready. I will fix some more issues soon.